app/views/smart_proxies/plugins/_puppet_ca.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="row">
+  <div class="col-md-12">
+    <div class="row">
+      <h4><%= _('Puppet CA Details') %></h4>
+    </div>
+    <%= render :partial => 'smart_proxies/plugins/plugin_version', :locals => {:feature => 'puppetca', :proxy => @smart_proxy } %>
+    <div class="row">
+      <div class="col-md-4">
+        <%= _('Hosts count') %>
+      </div>
+      <div class="col-md-8">
+        <%= link_to(@smart_proxy.puppet_ca_hosts.count, hosts_path(:search => "puppet_ca = #{@smart_proxy.name}")) %>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <%= _('Valid certificates') %>
+      </div>
+      <div class="col-md-8">
+        <% valid =  SmartProxies::PuppetCA.find_by_state(@smart_proxy, "valid").count rescue _('N/A') %>
+        <%= display_link_if_authorized(valid, hash_for_smart_proxy_puppetca_index_path(:smart_proxy_id => @smart_proxy, :state => 'valid').
+                                                          merge(:auth_object => @smart_proxy, :permission => 'view_smart_proxies_puppetca', :authorizer => authorizer)) %>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <%= _('Pending certificates') %>
```

Comment:
this looks nice, maybe we should highligh in a different color that there are pending certs?